notascript/src/GlobalStyles.js

```diff
@@ -0,0 +1,40 @@
+import { createGlobalStyle } from 'styled-components';
+
+export default createGlobalStyle`
```

Comment:
Hast du createGlobalStyle in der App.js importiert und eingebaut? Hab ich nichts zu gefunden 

GlobalStyle als Variablenname ist glaub ich die Konvention